lang/ilias_de.lang

```diff
@@ -5723,6 +5723,8 @@ common#:#tomorrow#:#Morgen
 common#:#toolbar_more_actions#:#Mehr Aktionen
 common#:#tools#:#Tools
 common#:#top_of_page#:#Seitenanfang
+common#:#tos_accept_usr_agreement_anonymous#:#Nutzungsvereinbarung
```

Comment:
IMO we should not use the `common` language module for this.

Reply:
Are you sure? e.g. `common#:#usr_agreement` and `common#:#usr_agreement_footer_intro` is also in common.